relay-metrics/src/aggregation.rs

```diff
@@ -1075,6 +1116,17 @@ impl CostTracker {
     }
 }
 
+impl fmt::Debug for CostTracker {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let mut values: Vec<(&ProjectKey, &usize)> = self.cost_per_project_key.iter().collect();
```

Comment:
If you collect this into a BTreeMap, you get sorting and also map display. That said, shouldn't `insta` take care of this when running the comparisons?

Reply:
> If you collect this into a BTreeMap, you get sorting and also map display.

good idea!

> That said, shouldn't insta take care of this when running the comparisons?

Apparently not, I ran into a flakey test after converting the map to `HashMap`.